addons/sale_project/views/project_update_template.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<odoo>
+
+    <template id="milestone_deadline_sale_project_inherit" inherit_id="project.milestone_deadline">
+        <xpath expr="//t[contains(@t-if, 'deadline')]" position="before">
+            <t t-if="milestone['quantity_percentage'] &gt; 0">
+                <t t-if="milestone['sale_line_name']">
+                    <span class="text-muted">(<t t-out="milestone['sale_line_name']"/> - <t t-out="100 * milestone['quantity_percentage']"/>%)</span>
+                </t>
+                <t t-else="">
+                    <span class="text-muted">(<t t-out="100 * milestone['quantity_percentage']"/>%)</span>
```

Comment:
```suggestion
                    <span class="text-muted">(<t t-out="(100 * milestone['quantity_percentage']).toFixed(2)"/>%)</span>
```
I think we need this to have `33,33%` i/o `33,3333333...%`, right?

Reply:
We round `quantity_percentage` from python and `toFixed` is js method so should I skip this? :slightly_frowning_face: